proto/definition/design_compose_definition.proto

```diff
@@ -24,7 +24,7 @@ option java_multiple_files = true;
 // A serialized Figma design document, containing views, images, component sets,
 // version, ID, and variable map. It's used for storing and loading Figma
 // designs in a DesignCompose project.
-message SerializedDesignDoc {
+message DesignComposeDefinition {
```

Comment:
Will you have a `DesignComposeDefinitionHeader` message?

Reply:
If we still need it, yes.